scheduler/cloudformation/template.go

```diff
@@ -54,6 +69,20 @@ func loadBalancerType(app *twelvefactor.Manifest, process *twelvefactor.Process)
 	return classicLoadBalancer
 }
 
+func schemeExposure(app *twelvefactor.Manifest, process *twelvefactor.Process) string {
+	env := twelvefactor.Env(app, process)
+	if v, ok := env["EMPIRE_X_EXPOSURE"]; ok {
```

Comment:
I think we should move this check into https://github.com/remind101/empire/blob/master/releases.go#L456, where it can just set the appropriate `Exposure` on the `twelvefactor.Process`.

I've been intending on moving all of the `EMPIRE_X_*` checks from this file and into [releases.go](https://github.com/remind101/empire/blob/master/releases.go) so this would be a good start.